src/App.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './App.css';
+import BoxContainer from './components/BoxContainer';
+import { IoIosInformationCircleOutline } from "react-icons/io";
+import SearchBar from './components/SearchBar';
+import Attempts from './components/Attempts';
+import Guesses from './components/Guesses';
+import GameOver from './components/GameOver';
```

Comment:
(nitpick) What I like to do for imports (regardless of what language I'm using) is separate them into sections based off where they come from (and then alphabetically/by directory):
- section for external dependencies (not installed by default in this language/installed by one of our developers)
- section for default dependencies (built-in imports for the language)
- section for ones that this project created/wrote (imports for other components in this repo, etc)

so this format here would look like:

```suggestion
import { IoIosInformationCircleOutline } from "react-icons/io";

import React from 'react';

import Attempts from './components/Attempts';
import BoxContainer from './components/BoxContainer';
import GameOver from './components/GameOver';
import Guesses from './components/Guesses';
import SearchBar from './components/SearchBar';
import './App.css';
```